docs/package_nRF5_boards_index.json

```diff
@@ -98,7 +98,8 @@
             {"name": "RedBear Blend 2"},
             {"name": "RedBearLab BLE Nano"},
             {"name": "RedBearLab nRF51822"},
-            {"name": "Waveshare BLE400"}
+            {"name": "Waveshare BLE400"},
+            {"name": "Electronut Labs Bluey"}
```

Comment:
Please remove this line, it will be added when the changes for the board are part of a release. Thanks.